CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(uprotocol-cpp-ulink-zenoh LANGUAGES CXX)
```

Comment:
please open the PR against https://github.com/eclipse-uprotocol/up-client-zenoh-cpp and not my fork.

Also please make sure that your code is aligned to https://github.com/eclipse-uprotocol/up-client-zenoh-cpp